gnd/src/main/java/com/google/android/gnd/ui/editrecord/EditRecordFragment.java

```diff
@@ -127,7 +120,7 @@ public void onStart() {
   private void onRecordChange(Persistable<Record> record) {
     switch (record.state()) {
       case LOADING:
-        saveRecordButton.setVisibility(View.GONE);
+        // do nothing
```

Comment:
Nit: Capitalize first letter, finish with "."

Also, please mention why we do nothing. I assume it's because we're handling this in the ViewModel, and we only handle recordChange in both places because some logic still remains in the Fragment?